docs/CHILI-GraFx/guides/manage-group-access/index.md

```diff
@@ -0,0 +1,22 @@
+# User Group Access
+
+!!! Info "Access"
+	Access is the set of rules that defines the role a user gets in a certain environment.
+
+## Manage Access
+
+Click the "Access" tab, on the group page.
+
+![screenshot-full](ug013.png)
+
+Click "Manage Access" to add or revoke access to that group.
+
+![screenshot](ug014.png)
+
+Under Access, add or remove the Environments.
+
+At the bottom part, you'll see each environment listed in the access field. Add the role the group members will receive in that environment. By default, the "End User" role is assigned, and you can add additional roles if wanted, for each Environment.
+
+## Template Designer Seats
+
+You won't be able to give Template Designer Seat through a group assignment. You'll have to [assign the Seat](/CHILI-GraFx/users/template-designer/) to each individual user.
```

Comment:
When a user has at Template Designer Seat you need to invite this user to the environment so he can access these environments. A Template Designer Seat does not give the use by default access to all environments in the subscription.